raft/server.cc

```diff
@@ -495,13 +510,13 @@ future<add_entry_reply> server_impl::execute_add_entry(server_id from, command c
     if (from != _id && !_fsm->get_configuration().contains(from)) {
         // Do not accept entries from servers removed from the
         // configuration.
-        co_return add_entry_reply{not_a_leader{server_id{}}};
+        co_return add_entry_reply{transient_error{format("Server {} is not part of the configuration", from), false, {}}};
```

Comment:
Can we make this message more clear? Something like "Not exception an entry from {} because it is not part of the config"?